src/ui/server/helpers/summary-lists/your-buyer/company-or-organisation/index.test.ts

```diff
@@ -0,0 +1,129 @@
+
+const { YOUR_BUYER: FIELD_IDS } = INSURANCE_FIELD_IDS;
+
+const {
+  INSURANCE: {
+    INSURANCE_ROOT,
+    YOUR_BUYER: { COMPANY_OR_ORGANISATION_CHANGE },
+  },
+} = ROUTES;
+
+const {
+  COMPANY_OR_ORGANISATION: { NAME, ADDRESS, COUNTRY, REGISTRATION_NUMBER, WEBSITE, FIRST_NAME, LAST_NAME, POSITION, EMAIL, CAN_CONTACT_BUYER },
+} = FIELD_IDS;
+
+describe('server/helpers/summary-lists/your-buyer/company-or-organisation-fields', () => {
+  describe('generateCompanyOrOrganisationFields', () => {
+    const mockAnswers = mockApplicationBuyer;
+    const { referenceNumber } = mockApplication;
+
+    const addressObject = generateAddressObject(mockAnswers);
+    const contactDetailsObject = generateContactDetailObject(mockAnswers);
+
+    const expectedBase = [
+      fieldGroupItem({
+        field: getFieldById(FIELDS.COMPANY_OR_ORGANISATION, NAME),
+        data: mockAnswers,
+        href: `${INSURANCE_ROOT}/${referenceNumber}${COMPANY_OR_ORGANISATION_CHANGE}#${NAME}-label`,
+        renderChangeLink: true,
+      }),
+      fieldGroupItem(
+        {
+          field: getFieldById(FIELDS.COMPANY_OR_ORGANISATION, ADDRESS),
+          data: mockAnswers,
+          href: `${INSURANCE_ROOT}/${referenceNumber}${COMPANY_OR_ORGANISATION_CHANGE}#${ADDRESS}-label`,
+          renderChangeLink: true,
+        },
+        generateMultipleFieldHtml(addressObject),
+      ),
+      fieldGroupItem({
+        field: getFieldById(FIELDS.COMPANY_OR_ORGANISATION, REGISTRATION_NUMBER),
+        data: mockAnswers,
+        href: `${INSURANCE_ROOT}/${referenceNumber}${COMPANY_OR_ORGANISATION_CHANGE}#${REGISTRATION_NUMBER}-label`,
+        renderChangeLink: true,
+      }),
+      fieldGroupItem({
+        field: getFieldById(FIELDS.COMPANY_OR_ORGANISATION, WEBSITE),
+        data: mockAnswers,
+        href: `${INSURANCE_ROOT}/${referenceNumber}${COMPANY_OR_ORGANISATION_CHANGE}#${WEBSITE}-label`,
+        renderChangeLink: true,
+      }),
+      fieldGroupItem(
+        {
+          field: getFieldById(FIELDS.COMPANY_OR_ORGANISATION, FIRST_NAME),
+          data: mockAnswers,
+          href: `${INSURANCE_ROOT}/${referenceNumber}${COMPANY_OR_ORGANISATION_CHANGE}#${FIRST_NAME}-label`,
+          renderChangeLink: true,
+        },
+        generateMultipleFieldHtml(contactDetailsObject),
+      ),
+      fieldGroupItem({
+        field: getFieldById(FIELDS.COMPANY_OR_ORGANISATION, CAN_CONTACT_BUYER),
+        data: mockAnswers,
+        href: `${INSURANCE_ROOT}/${referenceNumber}${COMPANY_OR_ORGANISATION_CHANGE}#${CAN_CONTACT_BUYER}-label`,
+        renderChangeLink: true,
+      }),
+    ];
+
+    it('should return fields and values from the submitted data/answers', () => {
+      const result = generateCompanyOrOrganisationFields(mockAnswers, referenceNumber);
+
+      expect(result).toEqual(expectedBase);
+    });
+  });
+
+  describe('generateAddressObject', () => {
```

Comment:
Can we match the ordering of the actual file so the helper functions are before the main function